src/components/GetRooms.vue

```diff
@@ -36,30 +36,35 @@ const config = {
 const app = firebase.initializeApp(config);
 const auth = firebase.auth();
 const db = getFirestore(app);
+const roomsCol = collection(db, "rooms");
 
-async function getRooms(db) {
-  const roomsCol = collection(db, "rooms");
+async function getRooms() {
```

Comment:
Given `db` is also in scope I decided there's not much point in passing it to the function.

Reply:
Cool. This confused me at first, but now I can see how the logic works out that we no longer need to pass it in. `db` is only used inside the collection function which is now carried in with `roomsCol`. So it doesn't need to be passed in.